src/Controllers/ShowQueueMonitorController.php

```diff
@@ -89,17 +89,25 @@ public function collectMetrics(): Metrics
 
         $metrics = new Metrics();
 
-        $sqlTimestampDiffFunction = 'TIMESTAMPDIFF';
         $connection = DB::connection();
 
-        if ($connection instanceof SqlServerConnection) {
-            $sqlTimestampDiffFunction = 'DATEDIFF';
+        $expressionTotalTime = DB::raw('SUM(TIMESTAMPDIFF(SECOND, `started_at`, `finished_at`)) as `total_time_elapsed`');
+        $expressionAverageTime = DB::raw('AVG(TIMESTAMPDIFF(SECOND, `started_at`, `finished_at`)) as `average_time_elapsed`');
+
+        if ($connection instanceof DatabaseConnections\SqlServerConnection) {
+            $expressionTotalTime = DB::raw('SUM(DATEDIFF(SECOND, `started_at`, `finished_at`)) as `total_time_elapsed`');
+            $expressionAverageTime = DB::raw('AVG(DATEDIFF(SECOND, `started_at`, `finished_at`)) as `average_time_elapsed`');
+        }
+
+        if ($connection instanceof DatabaseConnections\PostgresConnection) {
+            $expressionTotalTime = DB::raw('SUM(EXTRACT(EPOCH FROM (`finished_at` - `started_at`)))) as `total_time_elapsed`');
+            $expressionAverageTime = DB::raw('AVG(EXTRACT(EPOCH FROM (`finished_at` - `started_at`)))) as `average_time_elapsed`');
```

Comment:
1. There is a ) too much
2. Not sure if it's a windows or a postgres thing, but using ` throws an error.
These 2 lines should probably be:

 ```
 $expressionTotalTime = DB::raw('SUM(EXTRACT(EPOCH FROM (finished_at - started_at))) as total_time_elapsed');
$expressionAverageTime = DB::raw('AVG(EXTRACT(EPOCH FROM (finished_at - started_at))) as average_time_elapsed');

```